evaluator.go

```diff
@@ -372,21 +372,21 @@ func (ev *Evaluator) Run(prune []string) error {
 	paramErrs := MultiError{Errors: []error{}}
 
 	errors.Append(ev.RunPhase(MergePhase))
+	paramErrs.Append(ev.RunPhase(ParamPhase))
+	if len(paramErrs.Errors) > 0 {
```

Comment:
That's the behavior we had before this change, and it is unaffected by this change.